src/@next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx

```diff
@@ -14,26 +14,26 @@ export const AttributeValuesChecklist: React.FC<IProps> = ({
   valuesShowLimitNumber = 5,
   onValueClick,
 }: IProps) => {
-  const [viewAllOptions, setViewAllOptions] = React.useState(!valuesShowLimit);
+  const [viewAllOptions, setViewAllOptions] = useState(!valuesShowLimit);
 
   return (
     <S.Wrapper>
       {title && <S.Header>{title}</S.Header>}
       {values &&
-        values.map((value, index) => {
-          if (!viewAllOptions && index > valuesShowLimitNumber - 1) {
-            return <></>;
-          }
-          return (
+        values.map((value, index) =>
+          !viewAllOptions && index > valuesShowLimitNumber - 1 ? (
+            <Fragment key={index} />
```

Comment:
Why is the empty fragment used here at all? Can't we remove this whole condition?

Reply:
U can ask the author.. [which is you](https://github.com/mirumee/saleor-storefront/blame/master/src/%40next/components/molecules/AttributeValuesChecklist/AttributeValuesChecklist.tsx#L25) :)
I've just fixed console error.